diff.txt

```diff
@@ -0,0 +1,879 @@
+directory 'bndtools/'
+directory 'bndtools/'
+directory 'bndtools/'
+directory 'bndtools/'
+
+
+blade.authenticator.shiro-1.0.0.jar
+*******************************************************
+
+
+blade.authfailure-1.0.0.jar
+*******************************************************
+
+
+blade.configurationaction-1.0.0.jar
+*******************************************************
+CHANGED    MANIFEST   <manifest>
+ ADDED      HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
+ REMOVED    HEADER     Require-Capability:osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://java.sun.com/portlet_2_0))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/aui))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/portlet))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/theme))",osgi.extender;filter:="(&(osgi.extender=jsp.taglib)(uri=http://liferay.com/tld/ui))",osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.7))"
```

Comment:
That is what I thought, like it was scanning the JSPs twice, maybe in the OSGI-OPT/src or something,  but that doesn't explain why the PORTLET_2_0 isn't duplicated in that case.  its only the Liferay namespaced taglibs that are dups